microshift_running_apps/microshift-operators-olm.adoc

```diff
@@ -0,0 +1,62 @@
+
+include::modules/olm-fb-catalogs-example.adoc[leveloffset=+2]
+include::modules/olm-fb-catalogs-guidelines.adoc[leveloffset=+2]
+include::modules/olm-creating-fb-catalog-image.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/cli_tools/opm-cli[`opm` CLI reference]
+* link:https://https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/operators/understanding-operators#olm-file-based-catalogs_olm-packaging-format[File-based catalogs]
+
+include::modules/olm-creating-catalog-from-index.adoc[leveloffset=+2]
+
+[id="microshift-using-olm-disconnected"]
+== Using OLM in disconnected clusters
+For {microshift-short} clusters that are installed on restricted networks, also known as _disconnected clusters_, Operator Lifecycle Manager (OLM) by default cannot access sources hosted on remote registries because those remote sources require full internet connectivity.
+
+You can enable your cluster to use OLM in a restricted network using a workstation that has full internet access. The workstation is used to prepare local mirrors of the remote sources, and push the content to a mirror registry. See xref:../microshift_install/microshift-deploy-with-mirror-registry.adoc#microshift-deploy-with-mirror-registry[Mirroring container images for disconnected installations].
+
+The following steps are required to enable OLM in restricted networks:
+
+* Use a workstation with full internet access to create and push local mirrors of the catalog content to a mirror registry.
+* Add a `CatalogSource` to the cluster so that the OLM catalog Operator can use the local catalog on the mirror registry.
+* Make sure that {microshift-short} is configured to run in the restricted network.
+
+//See xref:../microshift_install/microshift-deploy-with-mirror-registry.adoc#microshift-deploy-with-mirror-registry[Mirroring container images for disconnected installations], xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-in-rpm-ostree-offline-use[Embedding applications for offline use], and (link after merge of offline-doc PR) [Configuring networking settings for offline use] for details.
+
+After enabling OLM in a restricted network, you can continue to use your unrestricted workstation to keep your local catalog sources updated as newer versions of Operators are released. See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/operators/administrator-tasks#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for more information.
+
+[id="additional-resources_microshift-operators_{context}"]
+[role="_additional-resources"]
+== Additional resources
```

Comment:
@ShaunaDiaz same concern. here are ocp doc link. some content of the links are not for microshift.